content/docs/start/experiments.md

```diff
@@ -109,10 +109,10 @@ compares any number of experiments in one table:
 $ dvc exp show --no-timestamp \
                --include-params train.n_est,train.min_split
 ┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
-┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┃ white:**Experiment**    ┃ yellow:**avg_prec** ┃ yellow:**roc_auc** ┃ blue:**train.n_est**┃ blue:**train.min_split** ┃
```

Comment:
I think we'll want dvc-specific labels like `neutral`, `metric`, `param` instead of `white`, `yellow`, and `blue`. This way, writers don't have to memorize which color is which and we get the option to change the color in the future.

Reply:
This name change would also allow us to apply bold on all these headers, as they are in the table, without specifying it with `**markdown bold quotes**` since all params/metrics are headers and all headers are bold (if I understand correctly).

Reply:
> I think we'll want dvc-specific labels like `neutral`, `metric`, `param` instead of `white`, `yellow`, and `blue`.

I think we can (ab)use these tables for other visuals and having more color options may be needed in the future. I'd prefer color names because those are what we see in the output, a document writer may not always know if `acc` is a param or metric, but can see its background is blue or yellow. 

At least, could you make these aliases?

Reply:
Also, color background and text style should have different labels IMO. I don't want to ask for your help every time we have a style change in the output :)

Reply:
about colour names, I'd say `C0`, `C1`, `C2`, `C3` (as per `matplotlib` convention) except in our case mapping to Iterative's colour scheme (#13ADC7, #945DD5, #F46737, #945DD5)

Reply:
I added aliases for the colors(`neutral` or `white`, `metric` or `yellow`, `blue` or `param`), but kept the text and background styles separate for now.

Reply:
Is it possible to style the table elements without having to format them manually after copy-pasting from terminal? Even if it's not exactly the same as colored terminal output

Reply:
It is highly terminal dependent @jorgeorpinel. Some terminals copy the control codes, some don't and without control codes it's impossible to detect the background color.

Reply:
OK. So it's not possible?